src/Image/ImageFactory.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Image;
+
+use Imagine\Image\ImagineInterface;
+use Contao\CoreBundle\Adapter\AdapterFactoryInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Creates Image objects
+ *
+ * @author Martin Auswöger <martin@auswoeger.com>
```

Comment:
Not providing your e-mail address will help defending spam, because crawlers can find your e-mail address in the raw content of the file.